build.gradle.kts

```diff
@@ -79,9 +79,7 @@ subprojects {
     // subproject task as of today.
     //
     testImplementation(rootProject.libs.bundles.junit)
-    testImplementation(rootProject.libs.bundles.kotlin)
```

Comment:
I think we need to rebase here.

Reply:
I had opened this before you merged your PR. Rebased now 👍